### PR TITLE
Added Checkout page

### PR DESCRIPTION
**Pull request by saurabhchirde**
Feature added:

1. Added checkout page
2. Item overview.
3. Address selection
4. Payment mode selection.

checkout

### CODE REVIEW

File: src/Pages/Checkout/CheckoutPage.jsx [hunk 1]
```diff
@@ -1,7 +1,17 @@
+import CheckoutDetails from "../../Components/CheckoutPage/CheckoutDetails/CheckoutDetails";
```
Comment (PuneetP16) [inline src/Pages/Checkout/CheckoutPage.jsx:1]: Suggestion: You could add delivery charges in addition to total bill

File: src/Components/CheckoutPage/CheckoutDetails/OverviewCards/PaymentOverviewCard.jsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import {
```
Comment (PuneetP16) [inline src/Components/CheckoutPage/CheckoutDetails/OverviewCards/PaymentOverviewCard.jsx:1]: Make payment button is appearing only when GPay is selected. If it's not intended do check for the issue.

image

**Review by PuneetP16 — APPROVED** [review]
Check comments.

File: src/Components/CheckoutPage/CheckoutDetails/OverviewCards/PaymentOverviewCard.jsx (hunk 2 above)
Comment (aiman-ansari) [reply src/Components/CheckoutPage/CheckoutDetails/OverviewCards/PaymentOverviewCard.jsx:1]: Its working fine now.

**Comment by sushmita12345** [comment]
- Code is easily understandable.
- Very clean code.
- UI looks decent.
- Proper use of variable.

**Review by sushmita12345 — APPROVED** [review]

**Review by aiman-ansari — APPROVED** [review]
Your UI looks so good. Everything looks good to merge.

File: src/Pages/Checkout/CheckoutPage.jsx (hunk 1 above)
Comment (saurabhchirde) [reply src/Pages/Checkout/CheckoutPage.jsx:1]: Ok, will think about this.

File: src/Components/CheckoutPage/CheckoutDetails/OverviewCards/PaymentOverviewCard.jsx (hunk 2 above)
Comment (saurabhchirde) [reply src/Components/CheckoutPage/CheckoutDetails/OverviewCards/PaymentOverviewCard.jsx:1]: It's a feature. To proceed further for checking out, the user would need to select a delivery address and a payment method as also mentioned just below the final amount. 
Failure to select any of the options, the user will not be able to make payment.

**Pull request merged**